lib/ddtrace/auto_instrument.rb

```diff
@@ -1,4 +1,7 @@
 # typed: strict
+# Entrypoint file for auto instrumentation.
+#
+# This file's path is part of the @public_api.
```

Comment:
May be worth adding that the public API here is the path to this file, as it doesn't otherwise expose any other user-callable code.

Reply:
> May be worth adding that the public API here is the path to this file, as it doesn't otherwise expose any other user-callable code.

This is what I tried to convey with `This file's path is part of the @public_api.`. Do you think that's not clear?

Reply:
Uhhh 😰 I somehow misread this, I guess I read it as "This file is part of the @public_api" or something. Please ignore >_>